server/src/helpers/validateUserProfile.helper.ts

```diff
@@ -0,0 +1,44 @@
+import { validate } from 'class-validator';
```

Comment:
Please create "validators" folder and move the file there

Reply:
Done - acc5d71909f3cd9dbc7fbbea434599cd6313a000